src/PrestaShopBundle/PrestaShopBundle.php

```diff
@@ -60,5 +63,61 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new RouterPass(), PassConfig::TYPE_AFTER_REMOVING);
         $container->addCompilerPass(new OverrideTranslatorServiceCompilerPass());
         $container->addCompilerPass(new OverrideTwigServiceCompilerPass());
+        $this->addModuleEntityMappingsPass($container);
+    }
+
+    /**
+     * @param ContainerBuilder $container
+     */
+    private function addModuleEntityMappingsPass(ContainerBuilder $container)
+    {
+        $installedModules  = $container->getParameter('kernel.active_modules');
+
+        foreach ($this->getModulesPaths() as $modulePath) {
+            if (in_array($modulePath->getFilename(), $installedModules)
+                && file_exists($modulePath . '/config/doctrine.yml')
+            ) {
+                $config = Yaml::parse(file_get_contents($modulePath . '/config/doctrine.yml'));
```

Comment:
ParseException is not handled

Reply:
it's a WIP, I will "scream" on you later :laughing: